src/ui/server/controllers/insurance/your-buyer/your-buyer-details/index.ts

```diff
@@ -0,0 +1,49 @@
+import { PAGES } from '../../../../content-strings';
+import { Request, Response } from '../../../../../types';
+import api from '../../../../api';
+import { ROUTES, TEMPLATES } from '../../../../constants';
+import insuranceCorePageVariables from '../../../../helpers/page-variables/core/insurance';
+import { yourBuyerPageVariables } from '../../../../constants/field-ids/insurance/your-buyer';
+import { mapCountriesSelect } from '../../../../helpers/mappings/map-countries-select';
+import { yourBuyerDetailsValidation } from './validation/yourBuyerDetailsValidation';
+
+export const get = async (req: Request, res: Response) => {
+  if (!req.session.submittedData || !req.session.submittedData.insuranceEligibility) {
+    req.session.submittedData = {
+      ...req.session.submittedData,
+      insuranceEligibility: {},
+    };
+  }
+
+  const countries = await api.external.getCountries();
```

Comment:
Can we use the new API call that gets the countries from keystone API please instead of external?

```js
const countries = await api.keystone.countries.getAll();
```